examples/evaluators/golang/simple/main.go

```diff
@@ -72,57 +83,107 @@ func main() {
 
 	start := time.Now()
 
+	//proposedMatchIds  []string,
+	//overloadedPlayers map[string][]int
+	//overloadedMatches map[int][]int
+	//approvedMatches   map[int]bool
```

Comment:
I think this comment is unnecessary, IDEs will tell devs what the types are. This will go out of date very quickly.